src/main/java/prefs/UserConfig.java

```diff
@@ -0,0 +1,7 @@
+package prefs;
+
+/**
+ * Abstraction for user-defined settings.
+ */
+public class UserConfig {
+}
```

Comment:
I don't see how it could work like that. 
In any case, our policy is not to write code 'for future use'. We can add this class only when we need it.